jwst/lib/set_telescope_pointing.py

```diff
@@ -878,6 +878,12 @@ def update_wcs_from_telem(model, t_pars: TransformParameters):
         logger.warning('Calculation of S_REGION failed and will be skipped.')
         logger.warning('Exception is %s', e)
 
+    # If TARG_RA/TARG_DEC still 0/0 (e.g. pure parallels with no defined target),
+    # populate with RA_REF/DEC_REF values
+    if model.meta.target.ra == 0.0 and model.meta.target.dec == 0.0:
```

Comment:
I know that the odds of having real target values of exactly zero x zero are pretty slim, I wonder if it might be worth decreasing the chances of a false positive by also adding a check of the value of VISITYPE (model.meta.visit.type) to ensure that it has "PARALLEL" in it.

Reply:
Smart, Nadia suggested the same thing - I just forgot to push that update to the PR. Whoops!